stories/display.stories.tsx

```diff
@@ -90,4 +92,191 @@ storiesOf('Display', module)
         searchBoxPosition: 'end',
       }
     )
+  )
+  .add(
+    'Modal',
+    withPlayground(
+      ({ container, dropdownContainer }) => {
+        function App() {
+          const modalRef = useRef(null);
+          const inputRef = useRef(null);
+          const [isShowing, setIsShowing] = useState(false);
+
+          const toggleModal = useCallback(() => {
+            if (isShowing) {
+              setIsShowing(false);
+              return;
+            }
+
+            setIsShowing(true);
+            setTimeout(() => {
+              if (inputRef.current) {
+                inputRef.current.focus();
+              }
+            }, 0);
```

Comment:
Is it the part you said it won't focus without setTimeout?

Reply:
Yes, using the `autoFocus` prop triggers this message:

```
Autofocus processing was blocked because a document already has a focused element.
```

Focusing at the end of the event loop works though.